prober/prober.go

```diff
@@ -323,9 +344,17 @@ func (p *Probe) run() (pi ProbeInfo, err error) {
 			p.recordEnd(err)
 		}
 	}()
-	timeout := time.Duration(float64(p.interval) * 0.8)
-	ctx, cancel := context.WithTimeout(p.ctx, timeout)
-	defer cancel()
+	ctx := p.ctx
+	if p.IsContinuous() {
+		p.mu.Lock()
+		p.lastErr = nil
```

Comment:
can we doc why this field needs to be nil when the probe is continuous?

Reply:
It turns out this was vestigial from a previous iteration of this change and is unnecessary. I've removed it.